packages/cmr-client/README.md

```diff
@@ -244,11 +23,6 @@ Cumulus is a cloud-based data ingest, archive, distribution and management proto
 
 Test with `npm run test`.
 
-
 ## Contributing
 
-See [Cumulus README](https://github.com/nasa/cumulus/blob/master/README.md#installing-and-deploying)
-
----
-
-Generated automatically using `npm run build-docs`
+[See our contributing guidelines](https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md)
```

Comment:
Nit: this should probably be a sentance, something like "To contribute, 
```suggestion
To make a contribution please [see our contributing guidelines](https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md).
```

Reply:
(and other instances below) - this is a personal style preference and is entirely your call.